readme.md

```diff
@@ -42,6 +42,45 @@ TBA
 
 ![db schema](https://github-production-user-asset-6210df.s3.amazonaws.com/19575839/261530664-6140d0e8-abf8-4fb8-80c9-ff9eb7adfcb6.png)
 
+```mermaid
+
+erDiagram
+
+    USERS {
```

Comment:
Let's name tables in a lower case, just like in the db?

Reply:
Done